uid2_client/token_generate_input.py

```diff
@@ -62,7 +62,7 @@ def _create_json_request_for_generate_token(property, value, tc_string, generate
         if tc_string is not None:
             json_object["tcf_consent_string"] = tc_string
         if not generate_for_opted_out:
-            json_object["policy"] = 1
+            json_object["optout_check"] = 1
```

Comment:
can you add a unit test for this change ?

Reply:
A test exists at test_publisher_client#test_integration_optout_generate_token that will make sure it stil works after this change.